qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -36,6 +36,25 @@ def needs_trimming(f):
     return f
 
 
+class MountDetails():
```

Comment:
@batrick 
In context to commit "qa/cephfs: backup mount object before running tests ", I believe it would be better to move this commit to a new PR and make it go through a more rigorous QA than usual since it potentially affects all of CephFS QA.

https://github.com/ceph/ceph/pull/32581/commits/e891aaf6cca1e41d9798b2dd0b78ca8880a67cf9

Reply:
Moving it out is okay with me.

Reply:
https://github.com/ceph/ceph/pull/36780

Labelling this PR with DNM until #36780 gets merged.

Reply:
The commit depends on QA changes in here, can't move it out and test it separately until the QA changes here are merged. Removing DNM.